pkg/template/depgraph.go

```diff
@@ -21,6 +23,17 @@ type depGraph struct {
 	CACertItemsFromKey  map[string]map[string]struct{} // items that use the TLSKeyFromCA template function. map of TLSKeyFromCA configoptions to the canames and certnames they use
 }
 
+// These functions will be used to figure out dependency order in the event standard rendering fails
+// The first argument to each is a string with the dependent config item.
+// TLS<blah> functions are deprecated and not included.
+const replFuncReExpr = `(?:ConfigOption|ConfigOptionIndex|ConfigData|ConfigOptionFilename|ConfigOptionEquals|ConfigOptionNotEquals)[[:blank:]]*\"[^\"]*\"`
```

Comment:
how about this? change summary:
- only match space not all blanks
- one or more spaces, not zero or more
- escaping quote shouldnt be necessary
- one or more not quote, not zero or more

```suggestion
const replFuncReExpr = `(?:ConfigOption|ConfigOptionIndex|ConfigData|ConfigOptionFilename|ConfigOptionEquals|ConfigOptionNotEquals) +"[^"]+"`
```